lib/lib.misc.php

```diff
@@ -998,7 +998,12 @@ function rejudging_finish($rejudgingid, $request, $userid = NULL, $show_progress
 	auditlog('rejudging', $rejudgingid, $request.'ing rejudge', '(start)');
 
 	while ( $row = $res->next() ) {
-		if ( $show_progress ) echo "s" . specialchars($row['submitid']) . ", ";
+		if ( $show_progress ) {
+			echo "s" . specialchars($row['submitid']) . ", ";
+			// Also flush the output, so the browser displays updates
+			ob_implicit_flush(true);
```

Comment:
Perhaps I don't understand the documentation correctly but why do we do this in every loop iteration?

Reply:
I think you are right that we can move this to above the loop